ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/DepositUtil.java

```diff
@@ -13,63 +14,149 @@
 
 package tech.pegasys.artemis.datastructures.util;
 
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.apache.tuweni.crypto.Hash;
+import tech.pegasys.artemis.datastructures.operations.Deposit;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static tech.pegasys.artemis.datastructures.Constants.DEPOSIT_CONTRACT_TREE_DEPTH;
+
 public class DepositUtil {
 
-  /**
-   * public static List<Deposit> generateBranchProofs(List<Deposit> deposits) { deposits =
```

Comment:
Do you just wanna delete these?

Reply:
No, this needs to be fixed. That's my next PR